clang/include/clang/Basic/Attr.td

```diff
@@ -2845,6 +2845,11 @@ def ReqdWorkGroupSize : InheritableAttr {
     ArrayRef<const Expr *> dimensions() const {
       return {getXDim(), getYDim(), getZDim()};
     }
+    bool isDependent() const {
```

Comment:
Should probably be `isValueDependent()` due to the other kinds of dependencies.

Reply:
done